### PR TITLE
update aspect ratio

### PR DESCRIPTION
**Pull request by rachelandrew**
Removing details of internal mapping and adding support only for property.

As discussed in #6786 this PR removes details of the internal mapping to HTML width and height, and instead adds the support for the property. Partial support behind a flag in Firefox, and support behind a flag in Chrome.

### CODE REVIEW

**Comment by ddbeck** [comment]
The `internal-value` feature ought to be replaced in the same PR as one that fixes #6786 (to ensure it's all fixed at once and reflected in the release notes). I'll open a complementary PR for this soon.

**Comment by ddbeck** [comment]
@rachelandrew I've opened #6918 to relocate the HTML data. If you restore the `internal-value` data in this PR, then I can track the move for the release notes a little more gracefully. Alternatively, I've requested your review on that PR—if we merge that one first, you can rebase this PR on top of it.

**Comment by rachelandrew** [comment]
@ddbeck sorry I don't understand what the end result is that you want. If you just want it put back then we can close this PR and start over?

File: css/properties/aspect-ratio.json [hunk 1]
```diff
@@ -53,92 +46,23 @@
...
+              "version_added": false
             }
           },
           "status": {
-            "experimental": true,
+            "experimental": false,
             "standard_track": true,
             "deprecated": false
           }
-        },
```
Comment (ddbeck) [inline css/properties/aspect-ratio.json:81]: Sorry, Rachel. I should've been more clear about what I was asking for. I'd like you to undo the changes from this line down. In other words, leave the `internal-value` feature in the file (which will be removed in my PR), but keep the changes from above this line.

**Review by ddbeck — CHANGES_REQUESTED** [review]

**Comment by rachelandrew** [comment]
@ddbeck hopefully what I've done is right, also added the note about the implementation of support for flex items in Firefox 83.

**Review by ddbeck — CHANGES_REQUESTED** [review]
One consistency suggestion here, but otherwise this looks really good. Thank you, @rachelandrew!

**Review by ddbeck — APPROVED** [review]
Looks good. Thank you!

To get the tests to pass (and to fix my earlier mistake of asking you to remove the subfeature removal), I'm going to push a merge from master into this branch, so it can appear in today's release. 🚀

**Comment by ddbeck** [comment]
Travis is stuck again. Merging without it.

**Pull request merged**